sbt/src/sbt-test/typelevel/scala-organization/build.sbt

```diff
@@ -0,0 +1,8 @@
+organization := "org.blah"
+
+//scalaOrganization := "org.scala-lang"
+scalaOrganization := "com.milessabin"
```

Comment:
I'm not sure. There could be, but do we really want a test to have that sort of external dependency?